### PR TITLE
[speech_recognition] add speech recognition package

### PR DESCRIPTION
**Pull request by furushchev**
this package can be used with no limit ( currently we must use Android or Chrome browser to recognize voice)
It should work after merge https://github.com/ros-drivers/audio_common/pull/60

### CODE REVIEW

File: speech_recognition/package.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<package>
+  <name>speech_recognition</name>
+  <version>0.0.1</version>
```
Comment (garaemon) [inline speech_recognition/package.xml]: fix version

File: speech_recognition/package.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  <description>The speech_recognition package</description>
+
+  <maintainer email="furushchev@jsk.imi.i.u-tokyo.ac.jp">Yuki Furuta</maintainer>
+
+  <license>MIT</license>
```
Comment (garaemon) [inline speech_recognition/package.xml]: We prefer BSD

File: speech_recognition/package.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  <author email="furushchev@jsk.imi.i.u-tokyo.ac.jp">Yuki Furuta</author>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>rospeex_core</build_depend>
+  <build_depend>rospeex_if</build_depend>
```
Comment (garaemon) [inline speech_recognition/package.xml]: It looks it does not require build_depends

**Comment by furushchev** [comment]
maybe `rospeex` is not yet released on jade

```bash
[speech_recognition] CMake Error at /opt/ros/jade/share/catkin/cmake/catkinConfig.cmake:75 (find_package): 

[speech_recognition]   Could not find a package configuration file provided by "rospeex_core" with 

[speech_recognition]   any of the following names:                              

[speech_recognition]                                                            

[speech_recognition]     rospeex_coreConfig.cmake                               

[speech_recognition]     rospeex_core-config.cmake                              

[speech_recognition]                                                            

[speech_recognition]   Add the installation prefix of "rospeex_core" to CMAKE_PREFIX_PATH or set 

[speech_recognition]   "rospeex_core_DIR" to a directory containing one of the above files.  If 

[speech_recognition]   "rospeex_core" provides a separate development package or SDK, be sure it 

[speech_recognition]   has been installed.                                      

[speech_recognition] Call Stack (most recent call first):                       

[speech_recognition]   CMakeLists.txt:4 (find_package)                          

[speech_recognition]                                                            

[speech_recognition]                                                            

[speech_recognition] -- Configuring incomplete, errors occurred!                

[speech_recognition] See also "/workspace/ros/ws_jsk_recognition/build/speech_recognition/CMakeFiles/CMakeOutput.log". 

[speech_recognition] See also "/workspace/ros/ws_jsk_recognition/build/speech_recognition/CMakeFiles/CMakeError.log". 

[speech_recognition] <== '/workspace/ros/ws_jsk_recognition/build/speech_recognition/build_env.sh /usr/bin/cmake /workspace/ros/ws_jsk_recognition/src/jsk_recognition/speech_recognition -DCATKIN_DEVEL_PREFIX=/workspace/ros/ws_jsk_recognition/devel -DCMAKE_INSTALL_PREFIX=/workspace/ros/ws_jsk_recognition/install' failed with return code '1' 

Failed   <== speech_recognition       [ 2.5 seconds ]   
```

**Comment by wkentaro** [comment]
@furushchev Add it to https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/.travis.rosinstall.jade.

**Comment by furushchev** [comment]
@wkentaro Thx added.

**Comment by furushchev** [comment]
Hmm. branch has conflicts... What should i do... ?

**Comment by wkentaro** [comment]
rebase?

**Comment by garaemon** [comment]
LGTM.
Add rospeex to jsk.rosinstall.jade in jsk_common in order to avoid https://github.com/jsk-ros-pkg/jsk_common/pull/1153

**Comment by furushchev** [comment]
@garaemon Thank you for reviewing.

> Add rospeex to jsk.rosinstall.jade in jsk_common in order to avoid jsk-ros-pkg/jsk_common#1153

Craeted PR. https://github.com/jsk-ros-pkg/jsk_common/pull/1199

**Comment by garaemon** [comment]
merge origin/master

**Comment by furushchev** [comment]
I Close this PR. see discussion https://github.com/jsk-ros-pkg/jsk_common/pull/1199

**Pull request closed without merging**